doc/guides/Developer-s Guide/plugin-development/event-notifications.md

```diff
@@ -378,7 +382,7 @@ Emitted after we've caught up to the chain head on first start. Lists all curren
 
 ```json
 {
-    "balance_snapshots": [
+    "balance_snapshot": [
```

Comment:
I can't see that you actually made this change?

Reply:
It is the typo in the documentation only. Event name and response both are `balance_snapshot` but document shows it as `balance_snapshots`